subprojects/core-api/src/main/java/org/gradle/api/cache/CacheResourceConfiguration.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.cache;
+
+import org.gradle.api.Incubating;
+import org.gradle.api.provider.Property;
+
+/**
+ * Represents the configuration of a given type of cache resource.
+ *
+ * @since 8.0
+ */
+@Incubating
+public
+interface CacheResourceConfiguration {
```

Comment:
Is there a reason for this newline? It seems odd.

Reply:
Good catch.  Pretty sure intellij did this.  Both of these classes were inner classes and I used an intellij refactor to pull them up to external classes.  For some reason, it inserted the newline there and I just didn't notice it.  🤷 

Fixed.